<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ООП игра</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Управление</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QPushButton" name="baseInfoButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Состояние базы</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="setLoggingTypeButton">
       <property name="text">
        <string>Настройки логирования</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="addUnitButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Добавить юнита</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="saveButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="startGameButton">
       <property name="text">
        <string>НАЧАТЬ НОВУЮ ИГРУ</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string>Загрузить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="FieldWidget" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>541</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="currentPlayerLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>280</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ход игрока №</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FieldWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">fieldwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
